### ISSUE TITLE
Access the onboard camera

### ISSUE THREAD

**Issue by mblees** [robotemi/sdk#387]
Hey there,
I want to develop a feature which allows me to use the camera to scan QR-Codes. It will be used to bring visitors to their desired location for a meeting.
Is there a functionality for accessing the feed from the camera or is there any other way to implement my problem?
I would be thankfull for any help.
Poirot

**Comment by zjn0505** [comment]
It is Android cameras, be accessed from Android native camera API.

**Comment by jay-m-dev** [comment]
@mblees were you able to build an app with this functionality?

**Comment by mblees** [comment]
> @mblees were you able to build an app with this functionality?

Yes i was able to use CameraX to access the Camera after a lot of issues while implementing

**Comment by vincenzomarotta** [comment]
> > @mblees were you able to build an app with this functionality?
> 
> Yes i was able to use CameraX to access the Camera after a lot of issues while implementing

Hi, can I contact you? I have to access temi's camera but I'm having some problems

**Comment by vincenzomarotta** [comment]
> @vincenzomarotta Feel free to add me on discord maybe i can help you out: rgfpoirot

Thanks 🙏🏻 I've sent you a request as lookatviki
